open-sphere-plugins/feature-actions/src/main/java/io/opensphere/featureactions/controller/StyleFactory.java

```diff
@@ -101,26 +92,26 @@ else if (action instanceof LabelAction)
                 }
             }
             StyleAction styleAction = (StyleAction)classToAction.get(StyleAction.class);
-            LabelAction labelAction = (LabelAction)classToAction.get(LabelAction.class);
+//            LabelAction labelAction = (LabelAction)classToAction.get(LabelAction.class);
 
             // The idea here is to create the style one first and then attach
             // the label settings to it, if possible
             if (styleAction != null)
             {
                 style = newStyleStyle(styleAction, dataType, defaultStyleCopy);
             }
-            if (labelAction != null)
-            {
-                containsLabelAction = true;
-                if (style == null)
-                {
-                    style = defaultStyleCopy;
-                }
-                style = addLabelStyle(labelAction, dataType, style);
-            }
+//            if (labelAction != null)
+//            {
+//                containsLabelAction = true;
+//                if (style == null)
+//                {
+//                    style = defaultStyleCopy;
+//                }
+//                style = addLabelStyle(labelAction, dataType, style);
+//            }

```

Comment:
I believe we need to keep this and I don't think this is the cause of the ticket.